src/clang_parser.h

```diff
@@ -3,6 +3,9 @@
 #include <clang-c/Index.h>
 #include "bpftrace.h"
 
+#define ASM_GOTO_WORKAROUND_H "asm_goto_workaround.h"
+#define ASM_GOTO_WORKAROUND_PATH "/bpftrace/include/" ASM_GOTO_WORKAROUND_H
```

Comment:
Should normally wrap the contents of a `#define` in brackets for preprocessor safety, but since `ASM_GOTO_WORKAROUND_PATH` is only used once anyway we might as well just get rid of this define. That would match how the other builtin headers are done too.